src/docs/product/error-monitoring/index.mdx

```diff
@@ -1,7 +1,29 @@
 ---
-title: Error Monitoring
+title: Issues
 sidebar_order: 1
-description: "Learn more about how the configuration of your SDK helps you resolve issues quickly in sentry.io."
+description: "Learn more about the Issues page and how you can use it to efficiently triage issues."
 ---
 
+The Issues page of [sentry.io](https://sentry.io) is where you can view, [triage](/product/error-monitoring/issue-details/), and resolve error events sent by your application. A typical application sends a large number of events to [sentry.io](https://sentry.io). To make them manageable, [sentry.io](https://sentry.io) groups similar events into issues based on the event stack trace and other information. You can think of an issue as a single bug or problem with your app. This grouping of events into issues allows you to see how frequently a problem is happening and how many users it's affecting. With this information, you can then make decisions about what to do.
+
```

Comment:
We should have a picture of the Issues page directly beneath the first paragraph